ports/zephyr/Kconfig

```diff
@@ -76,7 +76,7 @@ config MEMFAULT_SHELL
 
 config MEMFAULT_HTTP_ENABLE
         bool "Enables support for using Memfault's HTTP APIs"
-        default y if BOARD_NRF9160DK_NRF9160NS || BOARD_THINGY91_NRF9160NS
+        default y if SOC_SERIES_NRF91X && TRUSTED_EXECUTION_NONSECURE
```

Comment:
Yep, I'm doing this on purpose for consistency across boards. If an individual board wants to override the default, it can do that in board_defconfig or Kconfig.defconfig files.